src/screens/FeedScreen/Content.js

```diff
@@ -350,12 +352,12 @@ export const styles = StyleSheet.create({
     marginLeft: 8,
     marginRight: 8
   },
-  contentFeed: {
-    flex: 1,
+  contentFeed: (hasComment) => ({
+    flex: !hasComment ? undefined : 1,
     marginTop: 0,
-    height: '100%',
+    height: !hasComment ? dimen.normalizeDimen(355) : '100%',
     width: '100%'
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentFeed` style is now a function that takes `hasComment` as a parameter and conditionally sets the `flex` and `height` properties. This is a good approach for dynamic styling based on props. However, please ensure that the `dimen.normalizeDimen(355)` call returns a valid value for height.
```diff
+  contentFeed: (hasComment) => ({
+    flex: !hasComment ? undefined : 1,
+    marginTop: 0,
+    height: !hasComment ? dimen.normalizeDimen(355) : '100%',
+    width: '100%'
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->